state/executor.go

```diff
@@ -221,12 +219,18 @@ func (e *BlockExecutor) ApplyBlock(ctx context.Context, state types.State, block
 	if err != nil {
 		return types.State{}, nil, err
 	}
+	abciValUpdates := resp.ValidatorUpdates
+
+	validatorUpdates, err := cmtypes.PB2TM.ValidatorUpdates(abciValUpdates)
+	if err != nil {
+		return state, nil, err
```

Comment:
just to be consistent you can just return `types.State{}` instead of `state` for `err != nil` case

Reply:
Follow up issue: https://github.com/rollkit/rollkit/issues/1734